cli/services/helm.go

```diff
@@ -139,22 +139,42 @@ func (h *helm2_16) DeleteChart(chart string) error {
 	return nil
 }
 
-func (h *helm2_16) Init() error {
-	args := []string{
-		"init", "--namespace", "default",
+func (h *helm2X) Init() error {
+	args := []string{"create", "-n", "kube-system", "serviceaccount", "tiller"}
+	output, err := shell.Execute(".", "kubectl", args...)
+	if err != nil {
+		log.Error("Could not create Tiller ServiceAccount")
+		return err
+	}
+	log.WithFields(log.Fields{
+		"output": output,
+	}).Debug("Tiller ServiceAccount created")
+
+	args = []string{"create", "clusterrolebinding", "tiller", "--clusterrole=cluster-admin", "--serviceaccount=kube-system:tiller"}
+	output, err = shell.Execute(".", "kubectl", args...)
+	if err != nil {
+		log.Error("Could not create Tiller ClusterRoleBinding")
+		return err
 	}
+	log.WithFields(log.Fields{
+		"output": output,
+	}).Debug("Tiller ClusterRoleBinding created")
 
-	_, err := helmExecute(args...)
+	args = []string{
+		"init", "--wait", "--service-account", "tiller",
```

Comment:
Waiting to make sure the steps have everything in place.

The service account is needed to find the Tiller, but just for Helm 2.x